VEHICLE/SetVehicleEnveffScale.md

```diff
@@ -11,9 +11,18 @@ void SET_VEHICLE_ENVEFF_SCALE(Vehicle vehicle, float fade);
 
 Examples with a besra:
 
-- [fade value `0.0`](https://i.imgur.com/DXNk63e.jpg)
-- [fade value `0.5`](https://i.imgur.com/2Vb35fq.jpg)
-- [fade value `1.0`](https://i.imgur.com/aa8cxaD.jpg)
+- fade value `0.0`:
+
+![](./SetVehicleEnveffScale/CiGTTOw.webp)
```

Comment:
```suggestion
![image with fade value set to 0.0](./SetVehicleEnveffScale/CiGTTOw.webp)
```
Unsure what policies there are for accessibility on the dog but these were mostly here already so its probably better to keep them